tools/ci/Jenkinsfile

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/groovy
+
+// Copyright (C) 2019 Luxoft Sweden AB
+// SPDX-License-Identifier: MPL-2.0
+
+def image
+
+node("Component") {
+    stage("Clean up the environment") {
+        sh '''
+        rm -rf *
+        '''
+    }
+
+    stage("Checkout") {
+        checkout scm
+    }
+
+    stage("Build Docker container") {
+        image = docker.build("template-dbus-service", "--build-arg UID=\$(id -u) \
+                                                       --build-arg GID=\$(id -g) \
+                                                       -f ./tools/ci/Dockerfile .")
+    }
+
+    image.inside {
+        stage("Download Raspberry Pi SDK") {
+            sh '''
+            wget https://pelux.io/jenkins/job/pelux-manifests_NIGHTLY_rpi-qtauto/lastSuccessfulBuild/artifact/artifacts_rpi-qtauto/pelux-glibc-x86_64-core-image-pelux-qtauto-neptune-dev-cortexa7t2hf-neon-vfpv4-toolchain-*.sh/*zip*/artifacts_rpi-qtauto.zip
+            unzip artifacts_rpi-qtauto.zip
+            '''
+        }
+
+        stage("Install Raspberry Pi SDK") {
+            sh '''
+            chmod +x pelux-glibc-x86_64-*.sh
+            sudo ./pelux-glibc-x86_64-*.sh -y -d ~/opt/pelux-sdk-x86_64/
```

Comment:
You should be able to remove sudo now when installing to $HOME. Installed files will not be owned by root and can be purged without any permission problems...?

I am still not sure if this is done in an overlay2 fs or bind mount. Do you know? I guess ownership only matters if it is in bind mount... but we want a bind mount?

Reply:
If `sudo` is removed here `sudoers` stuff in Dockerfile can also be removed?